swagger.yaml

```diff
@@ -447,6 +447,35 @@ paths:
           description: "Invalid request body or unexpected Cognito response"
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /groups-report:
+    get:
+      tags:
+        - Groups
+      summary: "List groups and the membership of the groups"
+      description: "
+      If a preview team ( cognito group ) is empty of users then this is not on the report.  
+      If the preview team ( cognito group ) acts as a role (i.e. publishers, admin) then this is included on the report
+      if header accept "
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
```

Comment:
think you can add `text/csv` to this array here instead of the header parameter below.